src/backend/prisma/migrations/20230702055640_init/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateTable
+CREATE TABLE "project" (
+    "id" TEXT NOT NULL,
+    "name" VARCHAR(30) NOT NULL,
+    "description" VARCHAR(100) NOT NULL,
+    "url" VARCHAR(50) NOT NULL,
```

Comment:
Same for this, maybe one day you'll want to hash the name of the project URL or something along those lines.
let's say https://www.linkedin.com has 24 characters, already half down.

Reply:
I can always do another migration if ill need to, I think sometimes we try too much to solve future problems, and that gets things complicated (in general). For now, it seems fine, if ill encounter a problem in the future ill solve it then.